workloads/router-perf-v2/README.md

```diff
@@ -56,6 +56,7 @@ It's possible to tune the default configuration through environment variables. T
 | SAMPLES               | Number of samples to perform of each test | `2` |
 | HOST_NETWORK          | Enable hostNetwork in the mb client | `true` |
 | NUMBER_OF_ROUTERS     | Number of routers to test | `2` |
+| INGRESS_OPERATOR_IMAGE     | Image to use for the ingress-operator | `default image` |
```

Comment:
idk really what to put for the default here, it's just the default image

Reply:
Move it under `HAPROXY_IMAGE` (at the bottom) and put the default the same: "unset".

Also add the `INGRESS_OPERATOR_IMAGE` to the env.sh as I did with the `HAPROXY_IMAGE` (commented). A good public example, or something close to it, of the OPERATOR_IMAGE would be good. This way we're consistent.